bundles/org.openhab.binding.tplinksmarthome/src/main/resources/OH-INF/i18n/tplinksmarthome.properties

```
@@ -82,6 +82,8 @@ thing-type.tplinksmarthome.kp400.label = KP400
 thing-type.tplinksmarthome.kp400.description = TP-Link KP400 Smart Outdoor Plug
 thing-type.tplinksmarthome.kp400.group.outlet1.label = Outlet 1
 thing-type.tplinksmarthome.kp400.group.outlet2.label = Outlet 2
+thing-type.tplinksmarthome.kp401.label = KP401
+thing-type.tplinksmarthome.kp401.description = TP-Link KP401 Smart WiFi Outdoor Plug
```

Can you try to regenerate this file with `mvn i18n:generate-default-translations`? After that it will probably be synchronized to:
```suggestion
thing-type.tplinksmarthome.kp401.description = TP-Link KP401 Kasa Smart WiFi Outdoor Plug
```

---

Thanks for that info and instructions. Changes has been done and pushed. 

For future reference, I ran:
```
> mvn i18n:generate-default-translations -pl :org.openhab.binding.tplinksmarthome
```